trinity/protocol/les/normalizers.py

```diff
@@ -8,13 +8,22 @@
 from eth.rlp.headers import BlockHeader
 
 from trinity.protocol.common.normalizers import BaseNormalizer
+from trinity.rlp.block_body import BlockBody
 
 TResult = TypeVar('TResult')
 LESNormalizer = BaseNormalizer[Dict[str, Any], TResult]
 
 
+# Q: Shouldn't this be named GetBlockHeadersNormalizer?
```

Comment:
No, because it normalizes the response which is a `Blockheaders` message (responding to a `GetBlockHeaders` request).

Reply:
ah! makes sense